### ISSUE TITLE
abnormal mechanical nociception (MP:0002734)

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3095]
>If this is a request for a new term, please provide us with the following information.
>1. Preferred term label (e.g., asplenia)

a. increased mechanical nociception
b. decreased mechanical nociception

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a. increased capability to sense pain elicited by mechanical stimulation
b. increased capability to sense pain elicited by mechanical stimulation
pmid:25704616

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. & b. abnormal mechanical nociception (MP:0002734)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjwang

**Comment by sbello** [comment]
MP:0020955 - increased threshold 
MP:0020954 - decreased threshold
